storage/aptosdb/src/state_store/state_merkle_batch_committer.rs

```diff
@@ -62,6 +63,11 @@ impl StateMerkleBatchCommitter {
                         .state_merkle_db
                         .write_schemas(batch)
                         .expect("State merkle batch commit failed.");
+                    self.state_merkle_db
+                        .version_cache()
+                        .maybe_evict_version(&self.state_merkle_db.lru_cache());
+                    // TODO(grao): Consider remove the following sender once we verified the
+                    // version cache correctly cached all nodes we need.
```

Comment:
I assume currently the execution is the bottleneck, so you didn't bother to try this eagerly?

Reply:
Yes.